inference-engine/src/vpu/graph_transformer/include/vpu/middleend/allocator/allocator.hpp

```diff
@@ -126,6 +126,8 @@ class Allocator final {
 
     DataMap<allocator::MemChunk*> _memChunksPerData;
 
+    std::map<std::pair<DimVector, DimValues>, int> _staticShapeOffsets;
```

Comment:
Still don't understand why we need to take order into consideration

Reply:
Because if DimOrder = NWCH and DimValues = { N = 1, C = 10, H = 12, W = 16 } in blob DimValues will be written in order of DimOrder i.e 1 16 10 12

Reply:
It's still unclear why we need to look at order. Order of serialization/de-serialization of dimensions matters, but it should not depend on dims order

Reply:
In current version order of serialization/de-serialization has a direct dependence on DimOrder. If we will use only DimValues, we can face problem when 2 data with same ShapeOffset and different DimOrder are trying to read DimValues from blob and one will get right shapes, but another will read them in different order and will get wrong shapes.